Scripts/ComToTcpServer.py

```diff
@@ -115,20 +117,23 @@ def listen_named_pipe():
                         continue
 
                     write_log(False, data)
-                    if args.verbose:
-                        print(f"OUT: {data}")
+                    if args.output:
+                        text = data.decode('ascii', errors='replace')
+                        print(f"{text}")
                     out_queue.put(data)
 
                 while not in_queue.empty():
                     data = in_queue.get()
                     write_log(True, data)
-                    if args.verbose:
-                        print(f"IN: {data}")
+                    if args.output:
+                        text = data.decode('ascii', errors='replace')
+                        # prints with red color
+                        print("\033[31m" + text + "\033[0m")
                     win32file.WriteFile(handle, data, None)
 
         except pywintypes.error as e:
             if e.args[0] == 2:
-                if args.verbose:
+                if args.debug:

```

Comment:
Seems a little silly to have an argument to only have one print it controls, but I don't have a big problem with it